src/randoop/operation/SerializableConstructorCall.java

```diff
@@ -4,19 +4,24 @@
 import java.io.Serializable;
 import java.lang.reflect.Constructor;
 
-import randoop.util.Reflection;
-
+/**
+ *  Serializable version of {@link ConstructorCall} allowing tests to be
+ *  serialized. In particular, uses string representation of the constructor based on
```

Comment:
Thanks for this useful comment that explains what is different than the standard representation.